script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -111,47 +113,49 @@ moon deep in slumber.
 [sha:d902da20e395d22633f6f0f4c40b34074dd6ea15]{
 -- Page 44, Offset 30617.
 -- 突然、胸の古傷が痛んだ。
-Suddenly, I feel a pain from the scar in my chest.
+I feel a sudden pain in the scar on my chest.
```

Comment:
prefer "from the scar" since elsewhere he describes it as being the source of pain emanating out